closed/adds/jdk/src/share/native/jdk/crypto/jniprovider/NativeCrypto.c

```diff
@@ -38,6 +38,14 @@
 
 #define OPENSSL_VERSION_1_0 "OpenSSL 1.0."
 #define OPENSSL_VERSION_1_1 "OpenSSL 1.1."
+// needed for OpenSSL 1.0.2 Thread handling routines
```

Comment:
C-style comments must be used in C code (`/* ... */`). Please fix all occurrences.
It's surprising that this file compiles as is.